### DISCUSSION TITLE
Pooling models from "CART" imputed data, with zero-inflated negative binomial model

### DISCUSSION THREAD

**Discussion by ghost** [amices/mice#375] [Impute, analyse and pool]
Hello,

I did just have one other question from mice. If I'm running multiple imputation on a dataset with method "CART," and then I want to use a zero-inflated negative binomial model on the different sets using:

imp <- with(imp1, zeroinfl(       ))

It works, but when I go to pool the results, or even look at a summary of it, I get:

Error: No tidy method for objects of class zeroinfl (for summary(imp))
Error: No glance method for objects of class zeroinfl (for summary(pool(imp))

Just wondering if there's a work around to this? With the "with" statement working and me being able to produce different models on the different imputed sets, why couldn't I be able to go further and pool the results?

Thanks for all your help!

**Comment by gerkovink** [comment]
`mice` uses `broom` while pooling since version 3.0. 

See #121, #142, #148 and #199 for some more detail.

All the best,

Gerko

**Comment by stefvanbuuren** [comment]
See <https://broom.tidyverse.org/articles/adding-tidiers.html> how to create your own `glance` and `tidy` methods.

Already available methods:
- `broom`: <https://broom.tidyverse.org/articles/available-methods.html>
- `broom.mixed`: <https://github.com/bbolker/broom.mixed>
